docs/user_guides/client_installation/index.md

```diff
@@ -1,56 +1,106 @@
 
-## Hopsworks (including Feature Store and MLOps)
-The Hopsworks client library is required to connect to the Hopsworks Feature Store and MLOps services from your local machine or any other Python environment such as Google Colab or AWS Sagemaker. Execute the following command to install the full Hopsworks client library in your Python environment:
+## Hopsworks Python library 
+
+The Hopsworks Python client library is required to connect to the Hopsworks Feature Store and MLOps services from your local machine or any other Python environment such as Google Colab or AWS Sagemaker. Execute the following command to install the Hopsworks client library in your Python environment:
 
 !!! note "Virtual environment"
     It is recommended to use a virtual python environment instead of the system environment used by your operating system, in order to avoid any side effects regarding interfering dependencies.
 
 ```bash
-pip install hopsworks
+pip install hopsworks[python]
 ```
 Supported versions of Python: 3.8, 3.9, 3.10, 3.11, 3.12 ([PyPI ↗](https://pypi.org/project/hopsworks/))
 
-!!! attention "OSX Installation"
-    Hopsworks latest version should work on OSX systems without any additional requirements. However if installing an older version of the Hopsworks SDK you might need to install `librdkafka` manually. Checkout the documentation for the specific version you are installing.
+### Profiles
 
-!!! attention "Windows/Conda Installation"
+The Hopsworks library has several profiles that bring additional dependencies and enable additional functionalities:
 
-    On Windows systems you might need to install twofish manually before installing hopsworks, if you don't have the Microsoft Visual C++ Build Tools installed. In that case, it is recommended to use a conda environment and run the following commands:
-    
-    ```bash
-    conda install twofish
-    pip install hopsworks
-    ```
+| Profile Name       | Description   |
+| ------------------ | ------------- |
+| No Profile         | This is the base installation. Supports interacting with the feature store metadata, model registry and deployments. It also supports reading and writing from the feature store from PySpark environments. |
+| `python`             | This profile enables reading and writing from/to the feature store from a Python environment |
+| `great-expectations` | This profile installs the [Great Expectations](https://greatexpectations.io/) Python library and enables data validation on feature pipelines |
+| `polars`             | This profile installs the [Polars](https://pola.rs/) library and enables reading and writing Polars DataFrames |
 
-## Feature Store only
-To only install the Hopsworks Feature Store client library, execute the following command:
+You can install all the above profiles with the following command:
 
 ```bash
-pip install hsfs[python]
-# or if using zsh
-pip install 'hsfs[python]'
+pip install hopsworks[python,great-expectations,polars]
```

Comment:
Is Polars not just an extension of the python profile?
Is this not enough:
pip install hopsworks[polars,great-expectations]

Reply:
Doesn't look like from the project configuration: https://github.com/logicalclocks/hopsworks-api/blob/0c93c2ab91b3962a1bc45633def1e2320eb0f1d7/python/pyproject.toml#L91 - we had to put polars on its own profile because on certain VMs the instructions set is not available and importing polars was crashing `import hopsworks`